data/sql/updates/pending_db_world/rev_1605128047204479800.sql

```diff
@@ -0,0 +1,142 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1605128047204479800');
+
+-- SUMMARY: We edit the old table, then we create the new table, then we transfer the rows from the old table to the new table, then we remove no longer used columns from the old table, then we add foreign keys, then we add the strings, then update the .reload command
+
+-- Add new column ID
+ALTER TABLE `access_requirement`
+ADD `id` tinyint unsigned NOT NULL COMMENT 'The dungeon template ID' AUTO_INCREMENT UNIQUE FIRST;
+
+-- Set new primary key
+ALTER TABLE `access_requirement`
+ADD PRIMARY KEY `id` (`id`),
+DROP INDEX `PRIMARY`,
+DROP INDEX `id`;
+
+-- New table
+CREATE TABLE `dungeon_access_requirements` (
+  `dungeon_access_id` tinyint unsigned NOT NULL COMMENT 'ID from dungeon_access_template',
+  `requirement_type` tinyint unsigned NOT NULL COMMENT '0 = achiev, 1 = quest, 2 = item',
+  `requirement_id` mediumint unsigned NOT NULL COMMENT 'Achiev/quest/item ID',
+  `requirement_hint` varchar(255) COLLATE 'utf8_general_ci' NULL COMMENT 'Optional msg shown ingame to player if he cannot enter with extra info',
+  `faction` tinyint unsigned NOT NULL DEFAULT 0 COMMENT '0 = Alliance, 1 = Horde, 2 = both',
```

Comment:
small note, i finally verified, in the core TEAM_ALLIANCE = 0 and horde = 1, so i didn't mess up here :P